### PR TITLE
zh-CN:site/zh-cn/tutorials/quickstart

### PR DESCRIPTION
**Pull request by butyuhao**

### CODE REVIEW

File: site/zh-cn/tutorials/quickstart/beginner.ipynb [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "The image classifier is now trained to ~98% accuracy on this dataset. To learn more, read the [TensorFlow tutorials](https://www.tensorflow.org/beta/tutorials/)."
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/beginner.ipynb]: 这个不需要翻译吗

Comment (butyuhao) [reply site/zh-cn/tutorials/quickstart/beginner.ipynb]: 怪了，我commit过了，可是这边一点都没修改

File: site/zh-cn/tutorials/quickstart/beginner.ipynb [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "nnrWf3PCEzXL"
+      },
+      "source": [
+        "下载并安装 TensorFlow 2.0 测试版包。将TensorFlow载入你的程序："
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/beginner.ipynb]:
```suggestion
        "下载并安装 TensorFlow 2.0 测试版包。将 TensorFlow 载入你的程序："
```

File: site/zh-cn/tutorials/quickstart/beginner.ipynb [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "现在，这个照片分类器的准确度已经达到98%。想要了解更多，请阅读[TensorFlow 教程](https://www.tensorflow.org/beta/tutorials/)。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/beginner.ipynb]:
```suggestion
        "现在，这个照片分类器的准确度已经达到 98%。想要了解更多，请阅读 [TensorFlow 教程](https://www.tensorflow.org/beta/tutorials/)。"
```

**Comment by Mr-Linus** [comment]
@googlebot rescan
LGTM

**Comment by Mr-Linus** [comment]
@butyuhao  You need to comment the PR and trigger Bot detection CLAs.

**Comment by butyuhao** [comment]
OK, Thanks!

**Comment by butyuhao** [comment]
Try to trigger the bot.

**Comment by Mr-Linus** [comment]
cc @lamberta

**Comment by butyuhao** [comment]
There is something wrong. We both have cla but the bot says we don't,LOL. @lamberta

**Comment by Mr-Linus** [comment]
I found that if the author of PR commits my Suggested change, the CLAs bot will set the cla tag to no even if we have already signed CLAs.
It turns out that my CLA is no problem: <https://github.com/tensorflow/docs/pull/820>

**Comment by lamberta** [comment]
That is correct. If you add a commit by someone else then CLAbot will [comment](https://github.com/tensorflow/docs/pull/859#issuecomment-514273156). I just have to manually verify the username or email.

However, it looks like @butyuhao signed the CLA with a gmail account and these commits are using a hotmail account. And the co-authored commit by @Mr-Linus is using the qq account, and you signed the CLA with a gmail account.
So the bot will get confused in these cases. See the GitHub guide for setting emails in your account and using `.gitconfig`: https://help.github.com/articles/setting-your-commit-email-address
Thanks

**Pull request merged**